packages/swapper/src/swappers/osmosis/OsmosisSwapper.ts

```diff
@@ -274,6 +274,9 @@ export class OsmosisSwapper implements Swapper<ChainId> {
         })
 
       ibcSellAmount = await pollForAtomChannelBalance(receiveAddress, this.deps.osmoUrl)
+      // delay to ensure all nodes we interact with are up to date at this point
+      // seeing intermittent bugs that suggest the balances and sequence numbers were sometimes off
+      await new Promise((resolve) => setTimeout(resolve, 3000))
```

Comment:
This feels a little hacky - is there a way we can instead check the response from the node to see if we need to retry it for fresher data?

Reply:
Not that I know of easily. Its possible we no longer need this now that we are pointing at datahub nodes (much more reliable) but I would feel safer with them in there because of some bizarre behavior we have seen

Reply:
On a different note, might be time to switch to node 16 to benefit from `timers/promises` so we can `await setTimeout()` directly, one less method pushed to the next tick.
https://nodejs.org/api/timers.html#timerspromisessettimeoutdelay-value-options